guide/english/javascript/regular-expressions-reference/index.md

```diff
@@ -53,9 +53,9 @@ a character followed by a b followed by a c.
 
 ### Testing for matches
 ```
-console . log (/ abc /. test (" abcde ") );
+console . log (/abc/. test (" abcde ") );
 // → true
-console . log (/ abc /. test (" abxde ") );
+console . log (/abc/. test (" abxde ") );
```

Comment:
There was no reason to change this test, because the original already correctly return false.

Reply:
You are right. It already returned false but for the wrong reason. I believe the example here is to show that the test is false because the string is 'abxde' and we search for 'abc'. Anyway is false so I will revert the change.
Also I will remove all the unnecessary spaces.
Thank you.